0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,252 +1,135 @@
 
-#### Transport Layer
+The timeout of each successive transfer is shorter than the previous one, giving each connector a window of time to deliver the fulfillment even if their outgoing transfer was executed at the last possible moment.
 
-The transport layer is responsible for two things:
+For more details on the flow, see the [Interledger Protocol specification](../0003-interledger-protocol/0003-interledger-protocol.md) and the [Interledger whitepaper](https://interledger.org/interledger.pdf).
 
-* Generating the condition
-* Encoding (e.g. encrypting) the application layer data
-
-There are currently two transport layer protocols:
+<span class="show alert alert-info">**Note:** Interledger only supports Universal mode as described in the whitepaper. Atomic mode can be used by adjacent subsets of participants in an Interledger payment if desired, but this is not part of the standard.</span>
 
-* [Pre-Shared Key (PSK)](#pre-shared-key-psk)
+### Error Recovery and Connector Risk
 
-    The sender and recipient agree upon a secret key in advance. The sender uses this key to generate the condition and encrypt the application data. The recipient uses the same key to generate the fulfillment and decrypt the application data.
+An ILP execution chain can be interrupted if a connector fails to deliver the condition fulfillment. In this case, the sender thinks the transaction failed and retries it using the same condition as the original transaction. If the retry goes through the same connector that failed the first time, that connector can claim the incoming funds without sending a second outgoing transfer.
 
-* [Interledger Payment Request (IPR)](#interledger-payment-request-ipr)
+However, if a connector fails to deliver the fulfillment and the payment is not retried through the same route, the failed connector loses money. This is the main risk connectors face in Interledger. Connectors manage this risk through a variety of strategies such as selecting reliable ledgers and appropriately setting the window of time required between transfer expiries.
 
-    The recipient generates the condition during the quoting phase, and the sender cannot know the fulfillment until the recipient uses it to execute the payment. This is useful for building non-repudiable application layer protocols.
+## Interledger Protocol Suite
 
-#### Interledger Layer
+The Interledger stack is separated into four layers:
 
-All Interledger transport protocols use the [Interledger Protocol (ILP)](#interledger-protocol-ilp) to communicate with connectors about transfer requests. This may include requesting a quote or requesting a transfer on another ledger.
-
-The Interledger layer defines a standard way to refer to ledgers, accounts and amounts. This is used in routing as well as to try and make quotes comparable.
+![Interledger architecture layers](../shared/graphs/interledger-model.svg)
```

Comment:
The SVG doesn't render because of https://github.com/github/markup/issues/556, we should either find a workaround or render a PNG file as well from the graph. The only issue with doing that is the node libraries that do that include PhantomJS, which is a pretty heavy dependency just for rendering 1-2 PNG files.